subscription-manager.spec

```diff
@@ -885,6 +893,10 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 
 # Despite the name similarity dbus-1/system.d has nothing to do with systemd
 %config(noreplace) %{_sysconfdir}/dbus-1/system.d/com.redhat.*.conf
+%if 0%{?suse_version}
+%dir %{_sysconfdir}/dbus-1
+%dir %{_sysconfdir}/dbus-1/system.d
```

Comment:
OK, I see why they are needed. One small request: can you please move these two `%dir` together with the other `%dir` for SUSE? See almost 200 lines above:
https://github.com/candlepin/subscription-manager/blob/e16ba11bea4cfcaa80478c907013f9c7d9ef9c28/subscription-manager.spec#L706-L710

Reply:
I will work on it. Thank you for the feedback.